shell/artifacts/Social/source/PostTimeRanker.js

```diff
@@ -0,0 +1,31 @@
+// @license
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+
+defineParticle(({Particle}) => {
+  return class PostTimeRanker extends Particle {
```

Comment:
just a suggestion, maybe the particle needs to also get some sort of sorting configuration:
order: ascending / descending
and the name of a field to sort by?
totally fine to do it later, of course.

Reply:
It's a good suggestion, will wait to see if we actually want a recipe to do this and how to hook to the UI.